changelog/unreleased/enable-privatelinks-capability.md

```diff
@@ -0,0 +1,5 @@
+Change: Enable privatelinks by default
```

Comment:
why is `change` always breaking?

Reply:
Because the classification in calens maps very well to semver like that:
- `Security` and `Bugfix` for `Patch` version increment
- `Enhancement` for `Minor` version increment
- `Change` for `Major` version increment

Has been used in all web related repos like that since at least 1,5 years now. If not being used like that I don't understand the difference between `Change` and `Enhancement` to be honest. 😅